contentgrid-spring-data-rest/src/main/java/org/springframework/data/rest/webmvc/ContentGridSpringDataRestConfiguration.java

```diff
@@ -52,6 +55,22 @@ public Object postProcessAfterInitialization(Object bean, String beanName) throw
         };
     }
 
+    @Bean
+    public BeanPostProcessor replaceQuerydslWithCollectionFilterParamPredicateResolver(ApplicationContext applicationContext) {
```

Comment:
Is this temporary?  It looks pretty wrong if you have to resolve to BeanPostProcessor-tricks for your own beans :grimacing: